packages/common/src/client/appeals-api-client.js

```diff
@@ -164,10 +165,97 @@ class AppealsApiClient {
 		return response.json();
 	}
 
+	/**
+	 * @param {string} caseReference
+	 * @returns {Promise<(LPAQuestionnaireSubmission)>}
+	 */
+	async getLPAQuestionnaire(caseReference) {
+		const endpoint = `${v2}/appeal-cases/${caseReference}/lpa-questionnaire-submission`;
+		const response = await this.#makeGetRequest(endpoint);
+		return response.json();
+	}
+
+	/**
+	 * @param {string} caseReference
+	 * @param {string} lpaCode
+	 * @returns {Promise<(LPAQuestionnaireSubmission)>}
+	 */
+	async postLPAQuestionnaire(caseReference, lpaCode) {
+		const urlParams = new URLSearchParams();
+		urlParams.append('lpa-code', lpaCode);
+		const endpoint = `${v2}/appeal-cases/${caseReference}/lpa-questionnaire-submission?${urlParams.toString()}`;
+		const response = await this.#makePostRequest(endpoint);
+		return response.json();
+	}
+
+	/**
+	 * @param {string} caseReference
+	 * @param {object} data
+	 */
+	async patchLPAQuestionnaire(caseReference, data) {
+		const endpoint = `${v2}/appeal-cases/${caseReference}/lpa-questionnaire-submission`;
+		const response = await this.#makePatchRequest(endpoint, data);
+		return response.json();
+	}
+
+	/**
+	 * @param {string} caseReference
+	 * @param {string} questionnaireId
+	 */
+	async getSubmissionDocumentUploads(caseReference, questionnaireId) {
+		const urlParams = new URLSearchParams();
+		urlParams.append('questionnaire-id', questionnaireId);
+		const endpoint = `${v2}/appeal-cases/${caseReference}/lpa-questionnaire-submission/documentUpload?${urlParams.toString()}`;
```

Comment:
As with `lpa-code`, not sure this is needed. If we've got case ref, there is a 1:1 with questionnaire.

Reply:
Agreed, taken this out